client/src/context/auth-context/AuthContext.tsx

```diff
@@ -1,6 +1,6 @@
 // The AuthContext provides a state, which indicates whether the user is authenticated or not
 
-import React, { createContext, useReducer } from "react";
+import React, { createContext, useReducer, useContext } from "react";
```

Comment:
`useContext` is not used.


```suggestion
import React, { createContext, useReducer } from "react";
```